ReactAndroid/src/main/java/com/facebook/react/modules/camera/CameraRollManager.java

```diff
@@ -279,6 +286,19 @@ protected void doInBackgroundGuarded(Void... params) {
         selection.append(" AND " + SELECTION_BUCKET);
         selectionArgs.add(mGroupName);
       }
+      if (!TextUtils.isEmpty(mAssetType)) {
```

Comment:
This can't be empty - correct? Otherwise the code below won't filter photos or videos. What would the method return then?